docs/new_branch_checklist.md

```diff
@@ -0,0 +1,147 @@
+## New Branch Checklist
+
+### Key
+
+| Code            | Description                | Example     |
+|-----------------|----------------------------|-------------|
+| `<branch>`      | New branch name            | `lasker`    |
+| `<Branch>`      | New branch name titleized  | `Lasker`    |
+| `<branch#>`     | New branch number          | `12`        |
+| `<nextbranch>`  | Next branch name           | `morphy`    |
+| `<NextBranch>`  | Next branch name titleized | `Morphy`    |
+| `<nextbranch#>` | Next branch number         | `13`        |
+| `<nextbranch>`  | Next branch name           | `morphy`    |
+| `<n+2branch>`   | Unnamed N+2 branch         | `N-release` |
```

Comment:
would it be more helpful if I have the n+2 branch name, so it is not "unnamed"?

Reply:
Yes

Reply:
The only place it's really used is here, which is stating when deprecations are expected to be removed.

https://github.com/ManageIQ/manageiq/blob/c43b2093199bb6942318c3102371b79c03f95473/lib/vmdb/deprecation.rb#L4